types.go

```diff
@@ -438,3 +438,22 @@ type Flowtable struct {
 	// deleting it. When adding a new object, this must be nil
 	Handle *int
 }
+
+// Counter represents named counter
+type Counter struct {
+	// Name is the name of the named counter
+	Name string
+
+	// Comment is an optional comment for the counter
+	Comment *string
+
+	// Packets represents numbers of packets tracked by the counter
+	Packets *uint64
+
+	// Bytes represents numbers of bytes tracked by the counter
+	Bytes *uint64
```

Comment:
Maybe clarify that these will always be filled in by ListCounters but can be `nil` when creating a new counter.